### ISSUE TITLE
File input broken.

### ISSUE THREAD

**Issue by NexxusDrako** [NexxusDrako/leagion-scorinator#8]
This ts the error message that comes up.

```
C:\Users\user\leagion-scorinator>java leagion MCC.lgn
Oh slani!
"ava.lang.NumberFormatException: For input string: "1
        at java.lang.NumberFormatException.forInputString(Unknown Source)
        at java.lang.Integer.parseInt(Unknown Source)
        at java.lang.Integer.parseInt(Unknown Source)
        at TLeague.valuenate(leagion.java:715)
        at TLeague.<init>(leagion.java:304)
        at leagion.main(leagion.java:1513)
```

This is the file I'm attempting to use.

```
=regions 1
=teams 8
=KPB
=autofixtures
=2legs
=file
=
/
0 Northeastern FC
0 Western FC
0 Northwestern FC
0 Southeastern FC
0 Southwestern FC
0 Eastern FC
0 Southern FC
0 Northern FC
```

I think valuenate has a bug, but I'll see if it works under Linux before doing anything. I've a funny feeling it's Windows related.

**Comment by NexxusDrako** [comment]
```
"ava.lang.NumberFormatException: For input string: "1
```

Judging by the fact the last quote is all the way on the left, I smell either Java glitch, or that it's not ending line properly

**Comment by NexxusDrako** [comment]
Fixed it.

Offending code before

```
  int valuenate(String s, int prev)
  {
    int result;
    if (prev==0)
    {
      result=s.indexOf(" ");
      result=Integer.parseInt(s.substring(result+1, (s.length()));
    }else
    {
      result=prev;
      Out.println("Duplicate statement: "+s);
    }
    return(result);
  }
```

What changed: 

result=Integer.parseInt(s.substring(result+1, (s.length()**-1**)));

**Comment by NexxusDrako** [comment]
I think it was making length too long. Heh.

**Comment by NexxusDrako** [comment]
Right, given the fact that line endings are wonky in Windows, I'm going to test the change in Linux. If it fails to work in Linux, I will give the Linux rules precedence.

**Comment by NexxusDrako** [comment]
Fix works under Ubuntu 14.04 LTS. Will start pull request now.
